### DISCUSSION TITLE
part-ii/2013-25/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#352] [Question comments]
Part II 2013 Coding and Cryptography: Paper 4, Section I, $4 \mathrm{H}$ 

https://questions.tripos.org/part-ii/2013-25/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The substring is either $111000$ or $000111$ (and these correspond to one-time pad $k_1 \dots k_7$ equal to either $1011010$ or $0100101$). 

</details>
